.github/BOTMETA.yml

```diff
@@ -135,12 +135,16 @@ files:
   $modules/cloud/linode/: $team_linode
   $modules/cloud/lxd/: hnakamur
   $modules/cloud/memset/: glitchcrab
-  $modules/cloud/misc/ovirt.py: &virt
+  $modules/cloud/misc/ovirt.py: &ovirt
     ignored: skvidal vincentvdk
-    keywords: [ kvm, libvirt, ovirt, proxmox, qemu, RHEV, RHV ]
+    keywords: [ oVirt, RHV, RHEV ]
+    labels: [ cloud, ovirt ]
+    maintainers: $team_ovirt
+  $modules/cloud/misc/proxmox.py: &virt
+    ignored: skvidal vincentvdk
+    keywords: [ kvm, libvirt, proxmox, qemu ]
     labels: [ cloud, virt ]
     maintainers: $team_virt
-  $modules/cloud/misc/proxmox.py: *virt
```

Comment:
Why is this removed ? Please leave it in.

Reply:
It's still there, see line 143 in my changes. But because in line 138 I've replaced "&virt" with "&ovirt", I have to define "&virt" here. Or am I missing something?

Reply:
Oh right, sorry for that :-)